homeassistant/components/workday/config_flow.py

```diff
@@ -76,10 +77,12 @@ def validate_custom_dates(user_input: dict[str, Any]) -> None:
         if dt_util.parse_date(add_date) is None:
             raise AddDatesError("Incorrect date")
 
+    cls: HolidayBase = getattr(holidays, user_input[CONF_COUNTRY])
```

Comment:
Why are we not using `country_holidays()`?

Reply:
It's not necessary if you already have a country's class. I avoid importing an extra stuff keeping the code simple and still readable. The `country_holidays` function is a wrapper that does pretty much the same. I'm going to revisit moving back to `coutnry_holidays` in a following PR w/ language support.

Reply:
Ok, let's proceed.